panic on run-prepared

I'm trying to get rkt running on kubernetes, right now I'm running into a crash in rkt itself.  I'm not 100% sure what everything that I'm seeing means, but I can make the crash happen without using kubernetes at all, so I'm raising an issue here.  It might be that something is specified improperly in the manifest below (I'm assuming it's the isolators, since that's what's in the crash), but regardless the error should be caught rather than crashing.

Easy to reproduce:
foo.manifest:
```javascript
{  
   "acVersion":"0.6.1+git",
   "acKind":"PodManifest",
   "apps":[  
      {  
         "name":"worker",
         "image":{  
            "id":"sha512-cda68312426746e287f69640c2bc4d1b65c2d33732de808735c983b133d83720"
         },
         "app":{  
            "exec":[  
               "/bin/hello"
            ],
            "user":"0",
            "group":"0",
            "environment":[  
               {  
                  "name":"KUBERNETES_PORT",
                  "value":"tcp://10.0.0.1:443"
               },
               {  
                  "name":"KUBERNETES_PORT_443_TCP",
                  "value":"tcp://10.0.0.1:443"
               },
               {  
                  "name":"KUBERNETES_PORT_443_TCP_PROTO",
                  "value":"tcp"
               },
               {  
                  "name":"KUBERNETES_PORT_443_TCP_PORT",
                  "value":"443"
               },
               {  
                  "name":"KUBERNETES_PORT_443_TCP_ADDR",
                  "value":"10.0.0.1"
               },
               {  
                  "name":"KUBERNETES_SERVICE_HOST",
                  "value":"10.0.0.1"
               },
               {  
                  "name":"KUBERNETES_SERVICE_PORT",
                  "value":"443"
               }
            ],
            "mountPoints":[  
               {  
                  "name":"default-token-sd82m",
                  "path":"/var/run/secrets/kubernetes.io/serviceaccount",
                  "readOnly":true
               }
            ],
            "ports":[  
               {  
                  "name":"worker-tcp-5000",
                  "protocol":"TCP",
                  "port":5000,
                  "count":0,
                  "socketActivated":false
               }
            ],
            "isolators":[  
               {  
                  "name":"resource/cpu",
                  "value":{  
                     "request":"100m"
                  }
               }
            ]
         }
      }
   ],
   "volumes":[  
      {  
         "name":"default-token-sd82m",
         "kind":"host",
         "source":"/var/lib/kubelet/pods/48c30aec-57fc-11e5-8f54-42010af00504/volumes/kubernetes.io~secret/default-token-sd82m"
      }
   ],
   "isolators":null,
   "annotations":null,
   "ports":[  
      {  
         "name":"worker-tcp-5000",
         "hostPort":0
      }
   ]
}
```

Fetch the container, if you are wary of running my container, feel free to use your own, just replace the hash in the above json with the appropriate id.  I can't image the container itself makes a difference (although I don't actually know).
`sudo rkt-v0.8.0/rkt fetch montage-generator.appspot.com/hello:0.0.1`

Now prepare it and run it:
```
sudo rkt-v0.8.0/rkt prepare --pod-manifest foo.manifest --insecure-skip-verify
2015/09/10 21:24:44 Preparing stage1
2015/09/10 21:24:44 Loading image sha512-cda68312426746e287f69640c2bc4d1b65c2d33732de808735c983b133d83720
2015/09/10 21:24:44 Writing pod manifest
5af875c7-caf9-4cbb-8a2e-6cdc63e5874c
```

Make sure to replace the id in the following statement with whichever one you got in the previous step:
`sudo rkt-v0.8.0/rkt run-prepared --mds-register=false --private-net 5af875c7-caf9-4cbb-8a2e-6cdc63e5874c`

kaboom:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x5b013f]

goroutine 1 [running, locked to thread]:
github.com/coreos/rkt/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/resource.(*Quantity).Value(0x0, 0x7fe2e81be200)
	/home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/resource/quantity.go:336 +0x1f
github.com/coreos/rkt/common/cgroup.addCpuLimit(0xc208032f80, 0xb, 0x10, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/common/cgroup/cgroup.go:47 +0x6f
github.com/coreos/rkt/common/cgroup.MaybeAddIsolator(0xc208032f80, 0xb, 0x10, 0x84a880, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/common/cgroup/cgroup.go:63 +0x122
main.(*Pod).appToSystemd(0xc20804c180, 0xc20805c6c0, 0x846700, 0xc2080bb990, 0x6, 0x0, 0x0, 0x0, 0x0)
	/home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/pod.go:256 +0x2a10
main.(*Pod).PodToSystemd(0xc20804c180, 0x0, 0xc2080bb990, 0x6, 0x0, 0x0, 0x0, 0x0)
	/home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/pod.go:387 +0x524
main.stage1(0x7fe2e81bdc60)
	/home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/init.go:559 +0x13a4
main.main()
	/home/alban/go/src/github.com/coreos/rkt/release-build/gopath/src/github.com/coreos/rkt/stage1/init/init.go:781 +0x49
```

versions:
```
sudo rkt-v0.8.0/rkt version
rkt version 0.8.0
appc version 0.6.1
```

--- a/common/cgroup/cgroup.go
+++ b/common/cgroup/cgroup.go
@@ -58,8 +58,15 @@ func addMemoryLimit(opts []*unit.UnitOption, limit *resource.Quantity) ([]*unit.
 	return opts, nil
 }
 
+// MaybeAddIsolator considers the given isolator; if the type is known
+// (i.e. IsIsolatorSupported is true) and the limit is non-nil, the supplied
+// opts will be extended with an appropriate option implementing the desired
+// isolation.
 func MaybeAddIsolator(opts []*unit.UnitOption, isolator string, limit *resource.Quantity) ([]*unit.UnitOption, error) {
 	var err error
+	if limit == nil {
+		return opts, nil
+	}
 	if IsIsolatorSupported(isolator) {
 		opts, err = isolatorFuncs[isolator](opts, limit)
 		if err != nil {

